### ISSUE TITLE
Decide on IRI for gators example ontology

### ISSUE THREAD

**Issue by hlapp** [phyloref/phyloref-ontology#54]
At present the ontology IRI for the gators example is defined as `http://phyloref.org/clade-ontology/gators.owl`. This was probably meant to anticipate the base URI of the Clade Ontology, but it suffers from the following problems:
- This IRI does not currently resolve and thus results in a 404. (_This is not an error, though it isn't following Linked Data good practices either. Perhaps not a major concern for an example file._)
- There is no real option to deploy it from this repository such that it would resolve. _The phyloref.org website is managed in a different repo, and the `phyloref.org/clade-ontology` URI path would likely be served by the https://github.com/phyloref/clade-ontology repo._
- The right place to maintain this is arguably though this repo, not the the `clade-ontology` repo, as it's primarily an example for how the Phyloref and TCAN ontologies work together to form logical definitions for phyloreference classes.

We have, I think, therefore the following options.
1. Do nothing. The example works as is. _It's not web-accessible at a memorizable phyloref-branded URI, though._
2. Deploy it to http://ontology.phyloref.org/gators.owl. This can be done easily and automatically from this repo. _This means the URI at which it's accessible, even if memorizable, is not the one as which it identifies._
3. Deploy it to http://ontology.phyloref.org/gators.owl **and** change its IRI accordingly. _This would lose the reference to the Clade Ontology. Is this an issue at all?_

@gaurav your thoughts?

**Comment by gaurav** [comment]
> 3. Deploy it to http://ontology.phyloref.org/gators.owl and change its IRI accordingly. *This would lose the reference to the Clade Ontology. Is this an issue at all?*

I don't think that's an issue -- it's just an IRI. If we decided to include these phylorefs in the Clade Ontology, we could give it a CLADO IRI there. These two ontologies would also differ in some ways: these are human-generated from the clade definition directly, while the CLADO phylorefs would be generated by phyx.js from the Phyx file. So I think changing the IRI should be fine.

I really like the idea of having a resolvable IRI, so I think we should take the opportunity to change the IRI to the URL we deploy it to (option 3). The only comment I have is whether a URL like http://ontology.phyloref.org/examples/gators.owl might make it clearer what kind of ontology this is.

**Comment by hlapp** [comment]
> The only comment I have is whether a URL like http://ontology.phyloref.org/examples/gators.owl might make it clearer what kind of ontology this is.

Good point. I might need to look into the Makefile to possibly extract the subdirectory automatically for deployment. But even if it has to be done by hand, this would probably happen infrequently enough to not be an issue (aside from possibly forgetting it).
